assets/stylesheets/components/_search-result.scss

```diff
@@ -0,0 +1,11 @@
+.search-result {
+  @apply px-6 py-5 grid grid-cols-3 gap-4;
+
+  &__title {
+    @apply text-sm font-medium text-gray-500;
+  }
+
+  &__value {
+    @apply mt-0 text-sm text-gray-900 col-span-2;
+  }
```

Comment:
```suggestion
  &__title,
  &__value {
    @apply text-sm;
  }

  &__title {
    @apply font-medium text-gray-500;
  }

  &__value {
    @apply mt-0 text-gray-900 col-span-2;
  }
```

Reply:
Fixed in 603340805ae9f64933a351c7dbf84bd1a98f7ef5